hbase-server/src/test/java/org/apache/hadoop/hbase/TestClientClusterMetrics.java

```diff
@@ -104,6 +104,7 @@ public void testDefaults() throws Exception {
         defaults.getLiveServerMetrics().size());
     Assert.assertEquals(origin.getMasterInfoPort(), defaults.getMasterInfoPort());
     Assert.assertEquals(origin.getServersName().size(), defaults.getServersName().size());
+    Assert.assertEquals(ADMIN.getRegionServers().size(), defaults.getServersName().size());
```

Comment:
Why add this?

Reply:
Used for testing the Admin#getRegionServers method